web/source/osk/oskView.ts

```diff
@@ -1031,7 +1037,7 @@ namespace com.keyman.osk {
      * Scope        Public
      * @param       {(boolean|number)=}      bShow     True to display, False to hide, omitted to toggle
      */
-     ['show'](bShow: boolean) {
+     ['show'](bShow?: boolean) {
```

Comment:
This bothers me -- why are we toggling visibility of the language menu from `showLanguageMenu()`?

Reply:
I... uh, this isn't the relevant spot to put that comment, so I'm a little confused as to what you're actually asking.  Unless I'm missing something?  But the contents of the tagged method don't refer to the language menu whatsoever, so I'm at a loss.

If you're referring to our toggle in the rotation manager:

https://github.com/keymanapp/keyman/blob/c66bfac43f63c829d6b7d226029f26951cd49021/web/source/kmwrotation.ts#L44-L50

That's simply pre-existing behavior.  Or, at least, it was, and we're now restoring it.

Reply:
Apologies. This comment should have gone on line 989:

https://github.com/keymanapp/keyman/blob/a328a38297f3b2fec42bf3a08d14cbd44dcb64a9/web/source/osk/oskView.ts#L986-L991

But I see that it is on a new menu, so it would always be showing it anyway. Eliminating the unintuitive toggle behaviour altogether for a missing parameter would be better ... but as it is a public API we probably can't do that.

Reply:
Well... for the language menu, it's been like this for a _while_.

https://github.com/keymanapp/keyman/blob/86965b57408706eef6126ff2846c7799044ef7c4/web/source/kmwosk.ts#L1723-L1729

This was part of #1496.

That said, note how that `show` function is different from the one noted (originally) above.  The main OSK itself has been documented and published with that toggle behavior... so... yeah.